server/src/main/java/org/apache/iotdb/db/query/aggregation/impl/FirstValueAggrFunc.java

```diff
@@ -151,4 +152,18 @@ public void calcAggregationUsingTimestamps(long[] timestamps, int length,
   public boolean isCalculatedAggregationResult() {
     return resultData.isSetTime();
   }
+
+  @Override
+  public void calculateValueFromChunkData(ChunkMetaData chunkMetaData)
+      throws QueryProcessException {
+    if (resultData.isSetTime()) {
+      return;
+    }
+
+    Object firstVal = chunkMetaData.getStatistics().getFirstValue();
+    if (firstVal == null) {
+      throw new QueryProcessException("chunkMetaData contains no FIRST value");
+    }
+    resultData.putTimeAndValue(0, firstVal);
```

Comment:
Why here just set the timestamp to 0? You can get the time through chunkMetaData.getStatistics().getStartTime().